package com.aresky.chatservice.exception;

public class ExceptionMessage {
    public static final String ACCOUNT_NOT_FOUND = "Account not found!";
    public static final String STAFF_NOT_FOUND = "Staff not found!";
    public static final String ONLINE_STAFF_NOT_FOUND = "Không tìm thấy nhân viên nào đang trực tuyến!";
    public static final String INVALID_ACCOUNT_ID = "Invalid accountId!";
    public static final String INVALID_STAFF_ID = "Invalid staffId!";
    public static final String INVALID_CUSTOMER_ID = "Invalid customerId!";
    public static final String INVALID_CUSTOMER_EMAIL = "Invalid customer email!";
    public static final String CONVERSATION_NOT_FOUND = "Conversation not found!";
    public static final String INVALID_CONVERSATION_ID = "Invalid conversationId!";
    public static final String INVALID_STAFF_EMAIL = "Invalid staffMail!";
    public static final String STAFF_ALREADY_EXISTS = "Staff already exists!";
    public static final String ACCOUNT_ALREADY_EXISTS = "Account already exists!";
    public static final String CUSTOMER_ALREADY_EXISTS = "Customer already exists!";
    public static final String STAFF_NOT_ONLINE = "This staff is currently offline!";
}
